infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -289,6 +289,19 @@ function ensure_staging_gcb() {
     ensure_project_role_binding "${project}" "${principal}" "roles/cloudbuild.builds.builder"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectCreator"
     ensure_gcs_role_binding "${bucket}" "${principal}" "objectViewer"
+
+    local sa_name="gcb-signer"
+    local sa_email="${sa_name}@${project}.iam.gserviceaccount.com"
+    color 6 "Ensuring ${sa_email} exists and can sign artifacts in project: ${project}"
+    ensure_service_account \
+      "${project}" \
+      "${sa_name}" \
+      "used by prow to sign artifacts for ${project}"
+
+    ensure_serviceaccount_role_binding \
+        "${sa_email}" \
+        "serviceAccount:${sa_email}" \
+        "roles/iam.serviceAccountTokenCreator"
```

Comment:
@ameukam would this be correct to have a `gcb-signer` service account for every GCB project?

Reply:
> @ameukam would this be correct to have a `gcb-signer` service account for every GCB project?

@saschagrunert Yes, we'll need a service account for every GCP project but also if we drop usage for the default service account for GCB, we'll need to add the IAM permissions needed to run the build jobs (currently the GCB service account carry a large list of permissions https://cloud.google.com/build/docs/cloud-build-service-account). A binding between a dedicated service account and the role `roles/cloudbuild.builds.editor` will be needed in addition to the role for container images signatures.

Reply:
Thank you for the input, I added the role to the `gcb-signer` service account as suggested.